app/helpers/admin/applications_helper.rb

```diff
@@ -1,2 +1,7 @@
 module Admin::ApplicationsHelper
+  def group_by_city_select(users)
+    users.group_by(&:city).each do |city, user|
```

Comment:
Unused block argument - `city`. If it's necessary, use `_` or `_city` as an argument name to indicate that it won't be used.